src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(...args) {
```

Comment:
The `adder` function is designed to accept a variable number of arguments with the rest parameter syntax. However, according to the task description, the `adder` function should be able to be called with only one argument at a time. The rest parameter syntax should be removed, and the function should be modified to accept a single argument.